xplique/wrappers/pytorch.py

```diff
@@ -0,0 +1,158 @@
+"""
+Module for having a wrapper for PyTorch's model
+"""
+import warnings
+
+import tensorflow as tf
+import numpy as np
+
+from ..types import Union, Optional, Tuple, Callable
+
+class TorchWrapper(tf.keras.Model):
+    """
+    A wrapper for PyTorch's model so that they can be used in Xplique framework
+    for most attributions method
+
+    Parameters
+    ----------
+    torch_model
+        A nn.Module PyTorch model
+    device
+        If we are on GPU or CPU
+    is_channel_first
+        A boolean that is true if the torch's model expect a channel dim and if this one come first
+    """
+
+    def __init__(self, torch_model: "nn.Module", device: Union['torch.device', str],
```

Comment:
Sorry for being "picky" about this, but having both single quotes and double quotes on the same line might not be the best idea, haha!

Ideally, double quotes everywhere, triple for the doc.